assembly/driver/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>pentaho</groupId>
+    <artifactId>pdi-dataservice-client-assemblies</artifactId>
+    <version>8.1.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>pdi-dataservice-driver-bundle</artifactId>
+  <version>8.1.0.0-SNAPSHOT</version>
+  <name>Pentaho Community Edition Project: ${project.artifactId}</name>
+  <description>PDI Data Services driver bundle</description>
+  <url>http://www.pentaho.com</url>
+  <dependencies>
+    <dependency>
+      <groupId>pentaho</groupId>
+      <artifactId>pdi-dataservice-client-plugin</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>pentaho</groupId>
+      <artifactId>pdi-dataservice-client-plugin-api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>pentaho-kettle</groupId>
+      <artifactId>kettle-core</artifactId>
+      <version>${pdi.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>pentaho-kettle</groupId>
+      <artifactId>kettle-engine</artifactId>
+      <version>${pdi.version}</version>
+    </dependency>
+  </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-assembly-plugin</artifactId>
```

Comment:
Can you use the existing profile instead? https://github.com/pentaho/maven-parent-poms/blob/master/pom.xml#L882-L909

Reply:
Not without also changing the profile... It doesn't allow to change the generated file name.

Reply:
It seems to me that it would be better to change the profile to allow overriding the file name. @joaormf please reach out to @pentaho/2-1b / @hbfernandes to verify this.

Reply:
why do you want to change the generated name? it has no effects on the deployed artifact.
rather than that it's not doing much, I'd recommend using the existing profile.

Reply:
Well I don't want... it was already like that. But I will just remove the plugin override, and it will be generated with the version.